site/content/integrate/marketplace-submissions/_index.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Contribute to the Marketplace"
+heading: "Contribute to the Marketplace"
+description: "If you’ve built an integration, playbook, or tool which supports Mattermost, you may want to consider contributing it to the community via the Mattermost Web Marketplace."
+weight: 100
+aliases: /extend/plugins/community-plugin-marketplace/
+---
+
+The [Mattermost web Marketplace](https://mattermost.com/marketplace/) supports discovery, installation, and updates of extensions of the Mattermost platform. If you have built a plugin, app, playbook, integration, or tool that helps other users get more from Mattermost, the web Marketplace is a great way to get feedback on your contribution and help make it more popular. Once your submission is accepted to the Web Marketplace, Mattermost will also send you swag!
+
+All contributions are eligible to be added to our web marketplace at mattermost.com/marketplace after a review from our team. Apps and plugins that have been thoroughly reviewed by Mattermost will be eligible for our in-product marketplace as well with some additional requirements. Our in-product Marketplace enables customers to install and manage their apps and plugins from directly within the product. 
+
+## Type of integrations to submit to the web marketplace
+
+- Apps
+- Plugins
+- Webhook configuration
+- Playbook templates
+- Utility tools, such as importers, command line interfaces, and scripts
+
+## How to submit your contribution for consideration for the web marketplace
+
+Please fill out the information in [this form](https://forms.gle/PE8kmfSuneP9GWnq8) to be reviewed by a member of the Mattermost team. 
+
+Every contribution will go through the following checklist before being added to the web marketplace at mattermost.com/marketplace:  
```

Comment:
Couldn't "mattermost.com/marketplace" just be a hyperlink on "web marketplace" (or however we're going to capitalize it)?